state/watcher/txnwatcher_test.go

```diff
@@ -344,65 +330,97 @@ func (s *TxnWatcherSuite) TestInsertThenRemove(c *gc.C) {
 }
 
 func (s *TxnWatcherSuite) TestDoubleUpdate(c *gc.C) {
```

Comment:
This test is redundant now. Before it was checking converged events. Now it just goes through all of them.